dev/prod_builders.json

```diff
@@ -118,7 +118,8 @@
          "name": "Linux web_benchmarks_html",
          "repo": "flutter",
          "task_name": "linux_web_benchmarks_html",
-         "flaky": false
+         "flaky": false,
+         "upload_metrics": true
```

Comment:
should the property be defined https://github.com/flutter/infra/blob/master/config/devicelab_config.star#L81?

Reply:
Ahh that makes sense looking at https://cs.opensource.google/flutter/cocoon/+/master:app_dart/lib/src/service/luci.dart;l=194?ss=flutter

Thanks Keyong